ribs/archives/_cvt_archive.py

```diff
@@ -82,17 +83,21 @@ class CVTArchive(ArchiveBase):
         dtype (str or data-type): Data type of the solutions, objectives,
             and measures. We only support ``"f"`` / ``np.float32`` and ``"d"`` /
             ``np.float64``.
+        custom_centroids (array-like): If passed in, this (cells, measure_dim)
+            array will be used as the centroids of the CVT instead of generating
+            new ones. In this case, ``samples`` will be ignored, and
+            ``archive.samples`` will be None. This can be useful when one wishes
+            to use the same CVT across experiments for fair comparison.
+        centroid_method (string): Pass in the following methods for
```

Comment:
Also mention that the centroid method is adapted from the GECCO paper (add a link to the paper similar to the arxiv link above).